db/schema.rb

```diff
@@ -11,14 +11,14 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160519231024) do
+ActiveRecord::Schema.define(version: 20160528194021) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "games", force: true do |t|
-    t.integer  "jedi_user_id"
-    t.integer  "sith_user_id"
+    t.integer  "jedi_user"
+    t.integer  "sith_user"
```

Comment:
I think this is incorrect.  We want the field to be `jedi_user_id` and `sith_user_id` as the integer field in the db, but not in the model association.